sql.go

```diff
@@ -77,8 +91,19 @@ func GetSQL(data Filter, config *SQLConfig, dbArr ...DBDriver) (string, []interf
 			return inSQL(name, data.Includes, db)
 		}
 
-		values := data.Condition.getValues()
-		switch data.Condition.Rule {
+		values := data.getValues()
+
+		var err error
+		if config != nil && config.Predicates != nil {
+			if pr, prOk := config.Predicates[data.Predicate]; prOk {
```

Comment:
In think it has sense to return an error if Predicate was defined in configuration but can't be resolved ( as query will return wrong data in such case anyway )